java/test/jmri/managers/AbstractProvidingManagerTestBase.java

```diff
@@ -68,7 +66,7 @@ public void testRegisterDuplicateSystemName(ProvidingManager<E> m, String s1, St
 
             // If the test is unable to provide a named bean, abort this test.
             JUnitAppender.clearBacklog(Level.WARN);
-            log.debug("Cannot provide a named bean", ex);
+            // log.debug("Cannot provide a named bean", ex);
```

Comment:
Why is this log statement commented out?

Reply:
As this test is being aborted with a test Pass rather than failing for not providing a Bean,
I couldn't see any added value in the debug statement.
Perhaps in future it would be better for individual classes to override this Test?

Reply:
Will this test pass if the bean is not provided? Note the `Assume.assumeTrue("..", false)` statement below this comment.

Reply:
Sorry, I missed that it was an assume and not an assert. But in these cases, it would still be good with the debug statement. If I create a test that extends AbstractProvidingManagerTestBase and that the test doesn't work, I want to know why. Which is why the debug statement is there. So I think the debug statement should stay in place.

Reply:
But if you prefer to not restore the debug statement, the PR can be merged as is.

Reply:
Marking WIP, I'll have another look at this tomorrow . . .